src/cli/engine.ts

```diff
@@ -111,14 +111,14 @@ class Engine {
 
       if (Array.isArray(tableOperations)) {
         const [middleware, transforms] = tableOperations;
-        const requiredTransformers = this.#requiredTransformers({
+        const requiredTransformers = this.requiredTransformers({
```

Comment:
Can we change name of this variable to something else? it is conflicting with the function name
```suggestion
        const identifiedTransformers = this.requiredTransformers({
```